source/NVDAObjects/UIA/winConsoleUIA.py

```diff
@@ -32,24 +33,39 @@ def __init__(self, obj, position, _rangeObj=None):
 
 
 class winConsoleUIA(Terminal):
+	STABILIZE_DELAY = 0.03
```

Comment:
Could you elaborate on how you decided to go with `0.03`

Reply:
Value taken from legacy console code.